user/admin.py

```diff
@@ -1,17 +1,16 @@
 from django.contrib import admin
 from django.contrib.auth.admin import UserAdmin
-from django.utils.translation import gettext, gettext_lazy as _
-from .models import User
+from .models import CustomUser
 
 # Register your models here.
 class CustomUserAdmin(UserAdmin):
     fieldsets = (
         (None, {'fields': ('username', 'password')}),
-        (_('Personal info'), {'fields': ('date_of_birth', 'email', 'phone')}),
-        (_('Permissions'), {
+        (('Personal info'), {'fields': ('date_of_birth', 'email', 'phone')}),
+        (('Permissions'), {
```

Comment:
```suggestion
        ('Personal info', {'fields': ('date_of_birth', 'email', 'phone')}),
        ('Permissions', {
```
関数ではないので冒頭の()は不要です！
13行目も同様です！